config/initializers/gds_api.rb

```diff
@@ -0,0 +1,3 @@
+require 'gds_api/base'
+
+GdsApi::Base.logger = Logger.new(Rails.root.join("log/#{Rails.env}.api_client.log"))
```

Comment:
Just a good ole text file.

The JSON logs aren't getting pulled into Kibana for this app yet anyway so no rush either :)